src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -416,10 +440,10 @@ protected BlockPattern createStructurePattern() {
             .build();
     }
 
+    @Nonnull
     @Override
-    public void renderMetaTileEntity(CCRenderState renderState, Matrix4 translation, IVertexOperation[] pipeline) {
-        super.renderMetaTileEntity(renderState, translation, pipeline);
-        Textures.MULTIBLOCK_WORKABLE_OVERLAY.render(renderState, translation, pipeline, getFrontFacing(), isActive);
```

Comment:
Do I understand correctly that this is not needed as it was used for rendering front side which is done differently?

Reply:
This override was accidentally removed, when I instead just meant to change the drawing of the front face overlay. I have re-added this override, and changed the front overlay render to correctly reference the new method.